src/components/Dashboard/Dashboard.js

```diff
@@ -54,6 +54,7 @@ export default class Dashboard extends React.Component {
       <section className='dashboard'>
         <header>Dashboard</header>
         {myData}
+        <TotalContributions contribs={{total_donations: 10000, spent: 9000}} />
```

Comment:
This is just a placeholder right?

Reply:
Sorry yeah that is a placeholder I forgot to delete... Will fix it and push back up